config/helm/aws-node-termination-handler/README.md

```diff
@@ -66,13 +64,34 @@ Parameter | Description | Default
 `webhookTemplate` | Replaces the default webhook message template. | `{"text":"[NTH][Instance Interruption] EventID: {{ .EventID }} - Kind: {{ .Kind }} - Description: {{ .Description }} - State: {{ .State }} - Start Time: {{ .StartTime }}"}`
 `webhookTemplateConfigMapName` | Pass Webhook template file as configmap | None
 `webhookTemplateConfigMapKey` | Name of the template file stored in the configmap| None
-`dryRun` | If true, only log if a node would be drained | `false`
 `enableScheduledEventDraining` | [EXPERIMENTAL] If true, drain nodes before the maintenance window starts for an EC2 instance scheduled event | `false`
-`enableSpotInterruptionDraining` | If false, do not drain nodes when the spot interruption termination notice is received | `true`
+`enableSpotInterruptionDraining` | If true, drain nodes when the spot interruption termination notice is received | `true`
+`enableSqsTerminationDraining` | If true, drain nodes when an SQS termination event is received | `true`
```

Comment:
Sorry I missed this-- the code (`enableSQSTerminationDrainingDefault`) and helm chart has this defaulting to `false`

Reply:
oops good catch!